langchain4j-coherence/src/test/java/dev/langchain4j/store/embedding/coherence/CoherenceEmbeddingStoreIT.java

```diff
@@ -0,0 +1,71 @@
+package dev.langchain4j.store.embedding.coherence;
+
+import com.oracle.bedrock.junit.CoherenceClusterExtension;
+import com.oracle.bedrock.junit.SessionBuilders;
+import com.oracle.bedrock.runtime.coherence.CoherenceClusterMember;
+import com.oracle.bedrock.runtime.coherence.options.ClusterName;
+import com.oracle.bedrock.runtime.coherence.options.LocalHost;
+import com.oracle.bedrock.runtime.coherence.options.RoleName;
+import com.oracle.bedrock.runtime.coherence.options.WellKnownAddress;
+import com.oracle.bedrock.runtime.java.options.IPv4Preferred;
+import com.oracle.bedrock.runtime.java.options.SystemProperty;
+import com.oracle.bedrock.runtime.options.DisplayName;
+import com.oracle.bedrock.testsupport.junit.TestLogsExtension;
+import com.tangosol.net.Session;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.embedding.onnx.allminilml6v2q.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+/**
+ * An extension of the LangChain4J {@link EmbeddingStoreIT} tests
+ * that use Coherence as an {@link EmbeddingStore}.
+ */
+public class CoherenceEmbeddingStoreIT
+        extends EmbeddingStoreIT {
```

Comment:
```suggestion
        extends EmbeddingStoreWithFilteringIT {
```
This test class can inherit from `EmbeddingStoreWithFilteringIT` and get all the required tests